aya-obj/src/obj.rs

```diff
@@ -316,10 +318,38 @@ impl FromStr for ProgramSection {
             "xdp" => Xdp {
                 name,
                 frags_supported: false,
+                cpumap: false,
+                devmap: false,
             },
             "xdp.frags" => Xdp {
                 name,
                 frags_supported: true,
+                cpumap: false,
+                devmap: false,
+            },
+            "xdp_cpumap" => Xdp {
+                name,
+                frags_supported: false,
+                cpumap: true,
+                devmap: false,
+            },
+            "xdp_devmap" => Xdp {
+                name,
+                frags_supported: false,
+                cpumap: false,
+                devmap: true,
```

Comment:
Same here re: uses of `/` as delimeter...
See the way we do this for SK_SKB programs as reference as we already split on `/`, which makes this kind of messy... at least until #413 lands so we no longer derive name from the ELF section.

Reply:
Thanks, fixed.